packages/mask/.webpack/config.ts

```diff
@@ -136,6 +137,20 @@ export function createConfiguration(rawFlags: BuildFlags): Configuration {
                         },
                     },
                 },
+                {
+                    test: /\.svg$/,
+                    use: [
+                        {
+                            loader: 'svgo-loader',
```

Comment:
please don't add loaders to convert non-js files into a js file.

and it seems like you doesn't use it, so you don't need to add this.

Reply:
We only refer svg with `new URL()`, in that way webpack won't convert it into JS but file.

It's to minify svg for release.